src/EditorFeatures/CSharpTest/Diagnostics/MakeMethodAsynchronous/MakeMethodAsynchronousTests.cs

```diff
@@ -685,7 +685,7 @@ public void M1()
 
         [Theory]
         [InlineData(0, "Task")]
-        [InlineData(1, "void", Skip = "https://github.com/dotnet/roslyn/issues/18396")]
+        [InlineData(1, "void")]
         [Trait(Traits.Feature, Traits.Features.CodeActionsMakeMethodAsynchronous)]
         [WorkItem(18307, "https://github.com/dotnet/roslyn/issues/18307")]
         public async Task AddAsyncInLocalFunctionKeepsTrivia(int codeFixIndex, string expectedReturn)
```

Comment:
:memo: If you add a `string initialReturn` parameter before `string expectedReturn`, you could reuse this test for all four cases:

```
[InlineData(0, "void", "Task")]
[InlineData(1, "void", "void")]
[InlineData(1, "int", "Task<int>")] // This case is new but I'm guessing it already works
[InlineData(1, "Task", "Task")] // This case is new and I'm guessing it fails
```